### DISCUSSION TITLE
Can we please have the ability to tell a train to turn around in a station?

### DISCUSSION THREAD

**Discussion by ArielCMathews** [voxeltycoon/issues#1024] [General]
In the menu where we can add orders to load, unload, refit, wait, etc. Can we please have the option to tell a train to turn around and go back out the way it came from that stop? I hate having to put down a bunch of waypoints to try to make a train take the more efficient back and forth path instead of what it wants to do, which always seems to be going in a long circle away from its actual destination. I want more control over the specific route a train takes, and maybe being able to tell it to turn around when it's done at a station would help. 

Along the same lines, I'd like to be able to give a train orders at waypoints as to which direction to go if there is more than one path. Basically, I just want more route control. In a perfect world, I'd literally be able to draw out the route I want it to take and override the AI, but I'm not sure if that's possible or if anyone else wants something like that.

*(edit to add: the same goes for the road vehicles, honestly. If I could set their paths so they don't all drive into each other all the time, it would be amazing! And it would give me another layer of planning and management.)

**Comment by mfgCode** [comment]
You can actually already order the train to turn around after the loading at the station.
